### ISSUE TITLE
Alternatives to "W3C Council" for adjudicating formal objections

### ISSUE THREAD

**Issue by michaelchampion** [w3c/w3process#391]
I told the AB I strongly objected to the W3C Council as a mechanism for resolving formal objections.   Here are some alternatives to explore given that the idea of a "Technical Director" / "Director Lite" has been explored and got little support.

1. Select a small group of people with the necessary expertise and neutrality to arbitrate objections / appeals  (I'll call that the "Directorate" just to use a different word than "Council", but Council is a reasonable term too). The current AB+TAG would be a good starting point, but this creates issues about recusing people with a vested interest in a particular issue, how to handle transitions, etc.  I'd suggest the Directorate be *appointed* by the AB, TAG, W3M, maybe Board, maybe AC, maybe some group TBD.  Each of those bodies may, by consensus or supermajority, appoint a member of the Directorate for (staggered somehow) 1-2 year terms. Their appointee may or may not be a current member of the appointing group.  If any of the bodies can't get consensus on an appointee, they don't get to appoint anyone. 

1a.  Perhaps the Directorate doesn't collectively consider every appeal, but appoints an Arbitrator for each issue that comes to them.  That person should be knowledgeable in the field, neutral on the particular issue, and willing to invest the time to research the matter and write up and analysis.  The Arbitrator may or may not be a member of the Directorate, e.g. perhaps a Chair, a former TAG or AB member, or even the Founding Director would agree to take on the necessary work and commit to make a neutral decision

1b.  Maybe the Directorate votes to accept or not the Arbitrator's decision, maybe the Arbitrator gets to decide.  

If for whatever reason consensus fails here, the status quo (WG decision, AC vote) stands.

2. Double down on the Consensus principle, remove language about some authoritative tie breaking by the Director, [or Council, Directorate, etc.].  

2a.  Adopt the OASIS model -- Do a Call for Consensus; if more than n [1?  2?  5%?] of the responding members vote "-1", then do a formal  supermajority vote [2/3 ???] on the issue.  That vote is final and binding.

2b. Adopt the TC 39 model (as I understand it anyway), or maybe Quaker Consensus https://blog.fsmn.org/2012/08/understanding-quaker-decision-making/ is a better term:  Keep discussing until relevant stakeholders can live with a proposal ; no consensus means no standard. 

2c.  Yes, Quaker Consensus can lead to paralysis; but you don't have to be as nice as the Quakers are -- some large supermajority of a group can "kick out" a member who is not making objections on principled grounds or repeatedly blocks consensus.

**Comment by mnot** [comment]
Some form of (1) seems about right to me; I don't think consensus-of-members is necessarily good for the Web, with the current Membership model -- in addition to the issues around deadlock, exhaustion, etc.

I'd rather it _not_ be selected by the AB+TAG; that puts this group at even further remove from the community. Some other selection mechanism is necessary, I think.

**Comment by michaelchampion** [comment]
I'm not sure if I was clear -- in my original proposal the TAG and AB each get to select a person for the Directorate, but so do other bodies in the Community.  But I'd be more comfortable with an approach that does not "put this group at even further remove from the Community".  Any ideas? Some variant of a NomCom maybe, where people with lots of credentials / karma in the community are selected to do the selecting?

**Comment by dwsinger** [comment]
We don't have any authority to appoint anyone; we can only nominate people who are willing to volunteer. Telling TAG and AB members that it's a (small, we hope) part of being on those bodies is reasonable. Why would anyone else volunteer for such a miserable task, putting themselves into the middle of a conflict that the team is unable to resolve?

**Comment by dwsinger** [comment]
Overall, I think we all envisage that the council will self-select a subset to research and recommend a specific issue, and then the full council will consider and hopefully approve the recommended answer. We could formalize this, or even go so far as to say that only the subset  votes on the approval of the recommended answer. But I don't see that as an improvement. Getting full AB+TAG review of the solution, and approval, is both a sanity check, and gives the decision full-body weight.

I am still unclear what problem you are trying to solve.

**Comment by michaelchampion** [comment]
Problems with the Council proposal I am trying to solve include:

1. It's too big, so ultimately it will operate by politics rather than expert judgment.  

2. Yes, it will self-select the people who are willing to do the work to research the issue and write up a recommendation.  But those people are more likely to be advocates for one side or the other in the issue at stake than be neutral experts.

3. Both the TAG and AB are elected via Meek STV , which is not an effective way to select neutral experts, it's designed to ensure representation from the major factions. 

4. The history of the AB -- even before STV -- shows little indication that it can reach true consensus on much of anything.  The best we could hope for with the proposed Council would be a majority vote for or against an appeal. Anything less than a strong supermajority would not be a credible resolution that "sticks" with the losing side.

5. Both the TAG and AB already have more work than they can handle.

Bottom line: The TAG and  AB  are not likely to be composed of people who are willing to do the hard work to research the facts and the reasoning behind an objection, to form a reasonably objective conclusion, and to "make it stick" in the community.  TimBL did do these things, so it's not unreasonable to expect other humans to do them.    The challenge is to find a mechanism to select one, or a small group of qualified people who can arbitrate occasional but complex and controversial issues that lead to appeals/objections.

**Comment by dwsinger** [comment]
My sense is that the Directorate _is_ what we have on the table; the final consensus is of the whole AB+TAG but yes, they should select a subset or even an individual to bring a recommendation. What we're asking is that that recommendation be vetted, approved, or sent back for more work, by the entire AB and TAG. I don't see any way to delegate that authority; it comes by virtue of being elected.

**Comment by michaelchampion** [comment]
Closing in favor of https://github.com/w3c/w3process/issues/457 - Instead of a W3C Council, the bodies that would have made up the Council each get to appoint one person to a Directorate.
